server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -864,6 +876,8 @@ public synchronized void close() throws IOException {
         toClose.add(injector.getInstance(GatewayService.class));
         toClose.add(() -> stopWatch.stop().start("transport"));
         toClose.add(injector.getInstance(TransportService.class));
+        toClose.add(() -> stopWatch.stop().start("gateway_meta_state"));
```

Comment:
In ES this is done later - after the the ThreadPool. Not sure if it matters?

Reply:
Good catch thank you!